openhcl/virt_mshv_vtl/src/processor/mod.rs

```diff
@@ -180,13 +181,15 @@ mod private {
         pub(crate) hv: Option<VtlArray<ProcessorVtlHv, 2>>,
         pub(crate) vp_info: &'a TargetVpInfo,
         pub(crate) runner: &'a mut ProcessorRunner<'b, T::HclBacking>,
-        pub(crate) backing_shared: &'a BackingShared,
     }
 
     pub trait BackingPrivate: 'static + Sized + InspectMut + Sized {
         type HclBacking: hcl::ioctl::Backing;
+        type Shared;
```

Comment:
it would be good to say what should go in `Shared` and what shouldn't.

Reply:
This is a pattern I added that John's just refactoring here. The idea is it's a place for partition-wide state that's backing specific. As opposed to having everything on the Partition itself inside Options. Agreed it could use a comment.